generators/entity-server/templates/src/main/java/package/common/patch_template.ejs

```diff
@@ -56,5 +56,5 @@ _%>
          <%_ } %>
       })<% } %><% if (dto !== 'no') { %>
    .map(<%= mapper %>::toDto)<% } %><% if (isService && !reactive) { %>
-   .get()<% } %>;
+   .orElse(null)<% } %>;
```

Comment:
It sounds strange to return null after dealing with an optional, no?

Maybe throw an Exception or something like that will be better.

Reply:
I'd prefer to return the optional directly, and in Resource, instead of having:

```java
Optional<ChipsDTO> result = Optional.ofNullable(chipsService.partialUpdate(chipsDTO));
```

We'll have:

```java
Optional<ChipsDTO> result = chipsService.partialUpdate(chipsDTO);
```

Reply:
Oh yea for sure, so far better.

Reply:
ok guys, let's do this